### ISSUE TITLE
When searching for a phone number and clicking on it, by default it opens in TrueCaller instead of the Phone app in my phone.



### ISSUE THREAD

**Issue by shankar-balasubramaniam** [MM2-0/Kvaesitso#315]

**Comment by shankar-balasubramaniam** [comment]
![Screenshot_2023-04-02-08-41-01-56_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://user-images.githubusercontent.com/22413426/229329107-8c659d57-10de-4c9c-ba43-5d336490e6cc.jpg)

![Screenshot_2023-04-02-08-41-04-92_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://user-images.githubusercontent.com/22413426/229329108-af0c6b18-a3c5-4330-aae2-fc0ceaaf6840.jpg)

**Comment by MM2-0** [comment]
Did you set your default phone app to TrueCaller?

**Comment by shankar-balasubramaniam** [comment]
No. I even checked the default phone app. It was set to the Google Phone app.

**Comment by exstntlmsnthrp** [comment]
You may need to change your phone settings for opening links. Technically, Android handles a phone number like a link to direct a specific app to open it. 

Under your phone settings you can find it this way:

Apps > Default apps > Opening links 

From there look at both apps. You may need to enable the Google phone app to open them and you may need to disable from opening them.
